### DISCUSSION TITLE
windows defender the bot as a trojan. is it safe to allow?

### DISCUSSION THREAD

**Discussion by Lunaty332** [jagrosh/MusicBot#994] [Troubleshooting]
windows defender says that the bot is a trojan and i'm afraid to allow it (taking the bot from an unknown person is not ok not knowing what can happen). i used this and let it stay open on my pc even when i sleep and now i just installed a new version of windows 10 and i've seen that the bot dissapeared from my pc and i wanted to install it and i got a message about it.. anyways i think it's all about the now version of windows defender but i want to be sure
the photo of what it says in the windows defender:
![image](https://user-images.githubusercontent.com/92596825/137526265-ec800055-00ba-43bd-9dab-e0ad40d6b210.png)
hope it's all fine ^^

**Comment by Sanduhr32** [comment]
Read #958 please.

**Reply by Lunaty332** [reply]
i readed it but i'm not using bitdefender, i use windows defender and i wanted to be sure about the windows defender too

**Reply by Sanduhr32** [reply]
Windows Defender was also mentioned in the comments. I personally sometimes have issues that my Defender flags discord cache...

**Comment by MichailiK** [comment]
Windows Defender does not detect [JMusicBot-0.3.6.jar (SHA-256 `1DCDDDB0E0030BE8D5652718D7D3980E41E29A7B4C8DDC3F0680771144BF7749`)](https://github.com/jagrosh/MusicBot/releases/download/0.3.6/JMusicBot-0.3.6.jar) downloaded from [here](https://github.com/jagrosh/MusicBot/releases/tag/0.3.6) for me. Have you downloaded JMusicBot from a different place?

**Reply by Lunaty332** [reply]
no that's the single location form where i install it every time, i think it's about the new windows 10 version cause before updating the wondows it was totally ok

**Reply by MichailiK** [reply]
I'm currently running Windows 10 21H1 (Build 19043.1288), is yours newer?
